### PR TITLE
Create hubs_privacy_notice.md

### PR DESCRIPTION
**Pull request by mafeld**

### CODE REVIEW

File: en/hubs_privacy_notice.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+__Search providers:__ You can search for images, GIFs, and 3D Models to share in Hubs. When you search, we will send your searches to supported third parties to fulfill the search. Mozilla does not store your search queries or the search results. We support the following providers:
+* [Tenor](https://tenor.com/legal-privacy)
+* [Sketchfab](https://sketchfab.com/privacy)
+* [YouTube](https://policies.google.com/privacy)
+* [Twitch](https://www.twitch.tv/p/legal/privacy-policy/)
+
+__Twitter:__ If you connect Twitter to Hubs, you can tweet and share 2D images from Hubs rooms. Any content you tweet will be shared with Twitter and published on the Twitter platform. You can see [Twitter’s Privacy Policy](https://twitter.com/en/privacy) for more information.
+__Discord:__ If you connect Discord to Hubs, we store access tokens and the server and channel IDs that have been connected. We will synchronize chat messages, room changes, 2D and 3D objects you create, and whether you join or leave with the connected Discord channel. Hubs does not log any synchronized messages. You can see [Discord’s Privacy Policy](https://discordapp.com/privacy) for more information.
```
Comment (peiying2) [inline en/hubs_privacy_notice.md:62]: Add a line break between paragraphs.

File: en/hubs_privacy_notice.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+__Location information.__ Mozilla receives your IP address from your Firefox account when you sign up for and use the service. We use the IP address in order to approximate your location because Mozilla VPN is currently only available in certain countries.
+
+__Payment information.__ When you subscribe to Hubs, you will send payment through one of our third-party payment providers: Stripe, Apple, PayPal, or Google Pay. Mozilla receives a record of your account (including your billing address and the last four digits of your payment method) and the status of your account’s subscription. Mozilla does not store your full payment details.
+Usage Information. We receive information about the rooms and users in your Hubs instance. For instance, we learn when you create and delete rooms, how many rooms are in your Hubs instance, and how many users are in each room. We also receive information about how much storage space your Hubs instance is using.
```
Comment (peiying2) [inline en/hubs_privacy_notice.md:46]: Add a linebreak between the paragraphs.

**Review by peiying2 — COMMENTED** [review]
Even though the formatting is somewhat inconsistent (a line-break between paragraphs, or after a heading), it doesn't affect the output. The preview looks fine.

File: en/hubs_privacy_notice.md (hunk 1 above)
Comment (mafeld) [reply en/hubs_privacy_notice.md:62]: fixed.

File: en/hubs_privacy_notice.md (hunk 2 above)
Comment (mafeld) [reply en/hubs_privacy_notice.md:46]: fixed.

**Pull request merged**